### ISSUE TITLE
Could u provide more details about finetune?

### ISSUE THREAD

**Issue by HaomingX** [jaechan-repo/muse_bench#7]
Hello, thanks for ur really good work for unlearning. 
![image](https://github.com/user-attachments/assets/272198d9-ec99-4940-bbfd-1f8d4342d1a8)
![image](https://github.com/user-attachments/assets/6383ad5f-60aa-4683-ab87-f7b2c9530e65)
I combined raw/forget.txt +raw/ retain1.txt +raw/retain2.txt for target_model training according to the method mentioned in the paper, but the model loss is difficult to reduce (set the parameters to the same as yours). I can't train a model like the target model you provided. I would like to know if you also mix data in such a simple way?

**Comment by sh-qiangchen** [comment]
> Hello, thanks for ur really good work for unlearning. ![image](https://github.com/user-attachments/assets/272198d9-ec99-4940-bbfd-1f8d4342d1a8) ![image](https://github.com/user-attachments/assets/6383ad5f-60aa-4683-ab87-f7b2c9530e65) I combined raw/forget.txt +raw/ retain1.txt +raw/retain2.txt for target_model training according to the method mentioned in the paper, but the model loss is difficult to reduce (set the parameters to the same as yours). I can't train a model like the target model you provided. I would like to know if you also mix data in such a simple way?

If we directly use the provided target model, do you know what tokenizer they use?
